build/purser-install.sh

```diff
@@ -24,16 +24,22 @@ cd $HOME/opt/purser-env
 # === Purser Controller ===
 
 # Get Cluster config file location
-read -p "Taking $HOME/.kube/config as your cluster's configuration (yes/no): " isDefaultConfig
-if [ "$isDefaultConfig" != "yes" ]
+read -p "Enter your cluster's configuration. (Leave blank for default $HOME/.kube/config): " readConfig
```

Comment:
configuration path?
>>Leave blank for default.
This is not needed. Default behavior is pressing enter command if it is same in any installation.